rules/privileged-rules.py

```diff
@@ -62,9 +69,15 @@ def main():
        '-p', '--privilege_file',
        help='File in which to write privilege rules.'
     )
+    parser.add_argument(
+       '-x', '--prefix',
```

Comment:
Under indented.  You might consider also fixing the block above.

Reply:
Ah. There's some chuff in this PR. It should have only been STIG rules and the script. I'll fix shortly.